IterableAppExtensions/IterableAppExtensionsHost/ViewController.m

```diff
@@ -0,0 +1,29 @@
+//
+//  ViewController.m
+//  IterableAppExtensionsHost
+//
+//  Created by Victor Babenko on 4/30/18.
+//  Copyright © 2018 Iterable. All rights reserved.
+//
+
+#import "ViewController.h"
+
+@interface ViewController ()
+
+@end
+
+@implementation ViewController
```

Comment:
What exactly is this view controller used for?

Reply:
This particular folder should be deleted, as it is a duplicate, but there is an empty host app for tests. Notifications don't work properly unless they're tested inside a host app.